### ISSUE TITLE
/usr/lib32/libvulkan.so.1.3.264 from bundle wine is incompatible with Core™ 2nd generation 

### ISSUE THREAD

**Issue by inferrna** [clearlinux/distribution#2966]
![Screenshot from 2023-09-26 08-14-31](https://github.com/clearlinux/distribution/assets/4245277/2e47a206-3afb-4439-88bc-9604e0515853)

1. Go to the nearest museum for Core™ 2nd generation CPU
2. Clone vulkano project from https://github.com/vulkano-rs/vulkano/
3. `rustup toolchain install  stable-i686-unknown-linux-gnu`
4. In examples `cargo +stable-i686-unknown-linux-gnu build --bin teapot --release` (remove stupid from CFLAGS and CXXFLAGS if build fails)
5. Run `../target/release/teapot`
You will see:
```Rust
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: LibraryLoadFailure(DlOpen { desc: "/usr/lib32/libvulkan.so.1: CPU ISA level is lower than required" })', examples/src/bin/teapot/main.rs:68:40
```
Same time 64-bit teapot is working fine
md5sum:
`3cc5985f24009efc284eb60e1815e1a1  /usr/lib32/libvulkan.so.1.3.264`

Actually, I don't need this teapot, but also unable to run any 32-bit game from steam using proton on my **Xeon E5-2680V2**. This teapot is just the most obvious way to see the actual reason of error.

**Comment by fenrus75** [comment]
yeah there's something not quite right in the compiler flags for 32 bit
stuff -- we're looking at how to fix that properly
